src/render/handle.rs

```diff
@@ -129,6 +129,30 @@ impl Handle {
         f(label_text);
     }
 
+    pub fn get_karma_style(&self, karma: KarmaLevel) -> &'static str {
+        // TODO replace these with inline data image URIs
+        match karma {
+            KarmaLevel::Zero => {
+                "background-image: url(http://www.wikidot.com/userkarma.php?u=8976177"
```

Comment:
The brackets aren't closed for all these image urls

Reply:
Gah. I'll fix, good catch.